### PR TITLE
Feature/list users

### PR DESCRIPTION
**Pull request by moraesagustina**

### CODE REVIEW

File: app/views/devise/registrations/new.html.haml [hunk 1]
```diff
@@ -8,7 +8,7 @@
           %p.error-message= resource.errors.full_messages_for(:email).join(', ')
       .field
         - if @minimum_password_length
-          - placeholder_text = "Password (#{@minimum_password_length} characters minimum)"
+          - placeholder_text = 'Password (#{@minimum_password_length} characters minimum)'
```
Comment (EnzoCozza) [inline app/views/devise/registrations/new.html.haml]: Does this work for string interpolation?

File: app/views/devise/shared/_links.html.haml [hunk 2]
```diff
@@ -12,5 +12,5 @@
   %br/
 - if devise_mapping.omniauthable?
   - resource_class.omniauth_providers.each do |provider|
-    = button_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), data: { turbo: false }
+    = button_to 'Sign in with #{OmniAuth::Utils.camelize(provider)}', omniauth_authorize_path(resource_name, provider), data: { turbo: false }
```
Comment (EnzoCozza) [inline app/views/devise/shared/_links.html.haml]: Same here

File: app/views/users/_user.html.haml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+%li#show-user
+  %a{href: user_profile_path(user.nickname), id: 'profile'}
+    - if user.profile_picture.attached?
+      = image_tag (user.profile_picture), alt: 'profile picture', class: 'profile-picture'
+    - else
+      = image_tag 'no_profile_picture.png', alt: 'no profile picture', class: 'profile-picture'
+    .content
+      .p
+        %span.name #{user.first_name} #{user.last_name}
```
Comment (EnzoCozza) [inline app/views/users/_user.html.haml]: You are doing something similar in your user profile sidebar. What do you think about a function in the user model that returns the user's full name?

File: config/initializers/kaminari_config.rb
Comment (EnzoCozza) [inline config/initializers/kaminari_config.rb:1]: Let's remove the unused ones!

File: Gemfile [hunk 4]
```diff
@@ -55,6 +55,8 @@ gem 'html2haml'
 
 gem 'devise', '~> 4.9'
 
+gem 'kaminari'
```
Comment (EnzoCozza) [inline Gemfile]: Let's add the gem's version you are using, so we don't get unexepcted updates later!

File: spec/requests/users_request_spec.rb
Comment (EnzoCozza) [inline spec/requests/users_request_spec.rb:1]: Let's add a test where you create 6 users, and verify that in the first page you get only 5, and in the second page you get the last one

File: app/views/devise/registrations/new.html.haml (hunk 1 above)
Comment (moraesagustina) [reply app/views/devise/registrations/new.html.haml]: I changed it in the edit_user branch, but it seems that I lost this change when I did the rebase!

File: app/views/users/_user.html.haml (hunk 3 above)
Comment (moraesagustina) [reply app/views/users/_user.html.haml]: rails best practices throw "remove unused methods (User#full_name)" even though I use the method in the views :/

**Review by EnzoCozza — APPROVED** [review]

**Pull request merged**